src/Microsoft.ApiDesignGuidelines.Analyzers/Microsoft.ApiDesignGuidelines.Analyzers.md

```diff
@@ -112,6 +122,26 @@ Severity: Warning
 
 Help: [http://msdn.microsoft.com/library/ms182159.aspx](http://msdn.microsoft.com/library/ms182159.aspx)
 
+### CA1028: Enum Storage should be Int32 ###
+
+An enumeration is a value type that defines a set of related named constants. By default, the System.Int32 data type is used to store the constant value. Although you can change this underlying type, it is not required or recommended for most scenarios.
+
+Category: Design
+
+Severity: Warning
+
+Help: [https://msdn.microsoft.com/en-us/library/ms182147.aspx](https://msdn.microsoft.com/en-us/library/ms182147.aspx)
+
+### CA1032: Implement standard exception constructors ###
+
+Failure to provide the full set of constructors can make it difficult to correctly handle exceptions.
```

Comment:
Perhaps specify the required set? Again, depending on whether formatting is allowed, this might not be so easy:

> Provide at least the following constructors for any type derived from `System.Exception`:
* `X()`
* `X(string message)`
* `X(string message, Exception innerException)`